official/vision/beta/projects/yt8m/yt8m_flags.py

```diff
@@ -0,0 +1,156 @@
+      " segment_labels). Otherwise, --train_input_path must be aggregated "
+      "video-level features. The model must also be set appropriately (i.e. to "
+      "read 3D batches VS 4D batches.")
+  flags.DEFINE_string(
+      "model", "LogisticModel",
+      "Which architecture to use for the model. Models are defined "
+      "in models.py.")
+  flags.DEFINE_bool(
+      "start_new_model", False,
+      "If set, this will not resume from a checkpoint and will instead create a"
+      " new model instance.")
+
+  # Training flags.
+  flags.DEFINE_integer(
+      "num_gpu", 1, "The maximum number of GPU devices to use for training. "
+      "Flag only applies if GPUs are installed")
+  flags.DEFINE_integer("batch_size", 1024,
+                       "How many examples to process per batch for training.")
+  flags.DEFINE_string("label_loss", "CrossEntropyLoss",
+                      "Which loss function to use for training the model.")
+  flags.DEFINE_float(
+      "regularization_penalty", 1.0,
+      "How much weight to give to the regularization loss (the label loss has "
+      "a weight of 1).")
+  flags.DEFINE_float("base_learning_rate", 0.01,
+                     "Which learning rate to start with.")
+  flags.DEFINE_float(
+      "learning_rate_decay", 0.95,
+      "Learning rate decay factor to be applied every "
+      "learning_rate_decay_examples.")
+  flags.DEFINE_float(
+      "learning_rate_decay_examples", 4000000,
+      "Multiply current learning rate by learning_rate_decay "
+      "every learning_rate_decay_examples.")
+  flags.DEFINE_integer(
+      "num_epochs", 5, "How many passes to make over the dataset before "
+      "halting training.")
+  flags.DEFINE_integer(
+      "max_steps", None,
+      "The maximum number of iterations of the training loop.")
+  flags.DEFINE_integer(
+      "export_model_steps", 1000,
+      "The period, in number of steps, with which the model "
+      "is exported for batch prediction.")
+
+  # Other flags.
+  flags.DEFINE_integer("num_readers", 8,
+                       "How many threads to use for reading input files.")
+  flags.DEFINE_string("optimizer", "AdamOptimizer",
+                      "What optimizer class to use.")
+  flags.DEFINE_float("clip_gradient_norm", 1.0, "Norm to clip gradients to.")
+  flags.DEFINE_bool(
+    "log_device_placement", False,
+    "Whether to write the device on which every op will run into the "
+    "logs on startup.")
+
+
+  # Eval flags
+  # flags.DEFINE_string(
+  #   "eval_input_path", "",
+  #   "File glob defining the evaluation dataset in tensorflow.SequenceExample "
+  #   "format. The SequenceExamples are expected to have an 'rgb' byte array "
+  #   "sequence feature as well as a 'labels' int64 context feature.")
+  flags.DEFINE_boolean("run_once", False, "Whether to run eval only once.")
+  flags.DEFINE_integer("top_k", 20, "How many predictions to output per video.")

```

Comment:
We usually don't define new flags for configs. It might be better to just replace these flags as fields in the config class. That way to can remove this file altogether and the users can specify these fields through --params_override or --config_file.